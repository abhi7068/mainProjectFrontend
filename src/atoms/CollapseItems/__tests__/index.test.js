import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import PrettoSlider from '../index';
import React from 'react';
afterEach(cleanup);
it('renders', () => {
  //   const data = {
  //     rating: 5,
  //     comment: 'Nice Picture',
  //     user: {
  //       name: 'Madan Murari',
  //     },
  //   };
  const props = {
    marksforPrice: [
      {
        value: 0,
        label: '0',
      },
    ],
    min: 0,
    max: 15000,
  };

  const { asFragment } = render(
    <PrettoSlider
      min={props.min}
      max={props.max}
      value={props.value}
      marks={props.marksforPrice}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
