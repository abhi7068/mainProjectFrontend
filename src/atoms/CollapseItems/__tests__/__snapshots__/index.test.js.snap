// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="makeStyles-margin-2"
    />
    <span
      class="MuiSlider-root WithStyles(ForwardRef(Slider))-root-3 MuiSlider-colorPrimary MuiSlider-marked"
    >
      <span
        class="MuiSlider-rail WithStyles(ForwardRef(Slider))-rail-8"
      />
      <span
        class="MuiSlider-track WithStyles(ForwardRef(Slider))-track-7"
        style="left: 0%;"
      />
      <input
        type="hidden"
        value="NaN"
      />
      <span
        class="MuiSlider-mark"
        style="left: 0%;"
      />
      <span
        aria-hidden="true"
        class="MuiSlider-markLabel"
        style="left: 0%;"
      >
        0
      </span>
      <span
        aria-labelledby="track-inverted-slider"
        aria-orientation="horizontal"
        aria-valuemax="15000"
        aria-valuemin="0"
        aria-valuenow="NaN"
        aria-valuetext="NaNÂ°C"
        class="MuiSlider-thumb WithStyles(ForwardRef(Slider))-thumb-4 MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-29"
        data-index="0"
        role="slider"
        tabindex="0"
      >
        <span
          class="PrivateValueLabel-offset-31 MuiSlider-valueLabel WithStyles(ForwardRef(Slider))-valueLabel-6"
        >
          <span
            class="PrivateValueLabel-circle-32"
          >
            <span
              class="PrivateValueLabel-label-33"
            >
              NaN
            </span>
          </span>
        </span>
      </span>
    </span>
    <div
      class="makeStyles-margin-2"
    />
  </div>
</DocumentFragment>
`;
